eagles-food-backend/appsettings.json

```diff
@@ -1,6 +1,6 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": "To be changed to prod db"
+    "DefaultConnection": ""
```

Comment:
need to locally ignore appsettings, try `git update-index --assume-unchanged appsettings.json`